hook.php

```diff
@@ -81,9 +81,9 @@ function plugin_treeview_install()
                   PRIMARY KEY (`id`)
                 ) ENGINE=InnoDB DEFAULT CHARSET={$default_charset} COLLATE={$default_collation} ROW_FORMAT=DYNAMIC;";
 
-        $DB->query($query) or die($DB->error());
+        $DB->doQuery($query) or die($DB->error());
```

Comment:
It is forbidden to `die()`
```suggestion
        $DB->doQuery($query);
```